LICENSE

```diff
@@ -26,9 +26,8 @@ Portions of pydicom (private dictionary file(s)) were generated from the
 private dictionary of the GDCM library, released under the following license:
 
   Program: GDCM (Grassroots DICOM). A DICOM library
-  Module:  http://gdcm.sourceforge.net/Copyright.html

```

Comment:
Removed this line as it is missing from the current GDCM [Copyright.txt](https://sourceforge.net/p/gdcm/gdcm/ci/master/tree/Copyright.txt) file.